### ISSUE TITLE
Add to Cart blank screen error

### ISSUE THREAD

**Issue by amityweb** [firesale/firesale#463]
Add to Cart gives me this:

[Thu Jan 15 14:55:30 2015] [warn] [client 2.102.115.89] mod_fcgid: stderr: PHP Fatal error:  Call to a member function num_rows() on a non-object in /home/xxxxxx/public_html/addons/shared_addons/modules/firesale/models/products_m.php on line 862, referer: http://www.xxxxxx.co.uk/category

Same error I was having on this issue https://github.com/firesale/firesale/issues/462

Is Firesale finished?

**Comment by zvineyard** [comment]
Hi @amityweb, I'm working on restarting FireSale as we speak. I'll look into your problem today.

**Comment by zvineyard** [comment]
@amityweb Which branch of the addon are you using? 1.2/develop or 1.2/master?

**Comment by amityweb** [comment]
If I go here https://github.com/firesale/firesale, then 1.2 develop shows as the default, so I originally downloaded this. When I noticed, I then also downloaded 1.1 master. Both gave me the issue.

**Comment by zvineyard** [comment]
I recommend trying it with 1.2/master. For some reason, the core devs changed the view structure in 1.2/develop to something different than what's in 1.2/master. You may need to uninstall FireSale, switch branches to 1.2/master, then reinstall. I was able to load /category without an issue on that branch.

**Comment by amityweb** [comment]
I installed 1.2/master, but still get issues. First I installed Firesale then immediately went to Firesale and saw this error http://oi58.tinypic.com/2z8vmoi.jpg 

I was still able to create a category, that was OK. 

When I went to create a product I first had a white screen when I saved it, then when I hit back to the products page in Admin, I see this issue http://oi58.tinypic.com/2pqrern.jpg

Note: In the modules section, it shows the version as 1.2.5-dev

I am using Pyrocms version is 2.2.2, Pro version.
